model/calendar_test.go

```diff
@@ -76,4 +76,11 @@ func TestUnitMapper(t *testing.T) {
 			So(calendar.FuncIsFilterDatePresent(), ShouldBeTrue)
 		})
 	})
+
+	Convey("FuncIsFilterCensusPresent should detect if census is checked", t, func() {
+		Convey("When census is not checked", func() {
```

Comment:
This only checks the default, and neglects the rest of the logic of the function.
Complete the test coverage for this function.